pages/ksphere/kommander/1.0/release-notes/index.md

```diff
@@ -27,6 +27,9 @@ enterprise: false
 
 - Add 1.16.8 to selectable Kubernetes versions.
 
+#### Known issues
+
+- If you've installed Kommander v1.0 on a Konvoy v1.4.3 cluster (or older), you will not be able to create managed clusters. It is recommended you upgrade to Kommander v1.1.0, and deploy the addon on a newer version of Konvoy (v1.5.0), to be able to create clusters.
```

Comment:
Oh you did take my suggestion. Okay how about this since we want to make it super clear.

```suggestion
- If you've installed Kommander v1.0 on a Konvoy v1.4.3 cluster (or older), you will not be able to create managed clusters. We recommend you upgrade to Kommander v1.1.0 and use Konvoy v1.5.0 (or newer), to be able to create clusters. Note: Kommander v1.1 is not supported on any version of Konvoy 1.4.
```